src/Components/Table/Table.scss

```diff
@@ -21,7 +21,7 @@ $table-row-background: $g3-castle;
 
 .cf-table--header-cell {
   color: $g15-platinum;
-  font-weight: 900;  
+  font-weight: 500;  
```

Comment:
Similar to what I mentioned about the transition from `700` to `500`. The leap from `900` to `500` seems like a drastic stylistic change. If the original intention of the font-weight was to have a bold appearance, shouldn't we maintain that here?